lib/common/client/middleware/response/mhv_errors.rb

```diff
@@ -15,6 +15,7 @@ class MHVErrors < Faraday::Response::Middleware
           #
           def on_complete(env)
             return if env.success?
+            return unless env[:body].is_a?(Hash)
```

Comment:
There's no point in proceeding if this has not been turned into a hash by earlier middleware, it will just raise a ` string not matched` type error.